triggers/service/src/test/resources/logback.xml

```diff
@@ -11,7 +11,7 @@
     <logger name="io.grpc.netty" level="WARN">
         <appender-ref ref="stderr-appender"/>
     </logger>
-    <logger name="com.daml.lf.engine.trigger" level="DEBUG">
+    <logger name="com.daml.lf.engine.trigger" level="INFO">
```

Comment:
happy to revert this but I found it super hard to find the right places with debug logs turned on.

Reply:
Seems fine to change